src/barfTurn.ts

```diff
@@ -296,7 +296,7 @@ const turns: AdventureAction[] = [
       const locationChoice =
         familiarChoice === $familiar`Space Jellyfish`
           ? $location`Pirates of the Garbage Barges`
-          : determineDraggableZoneAndEnsureAccess("backup");
+          : determineDraggableZoneAndEnsureAccess("yellow ray");
```

Comment:
let's go ahead and nix the jellyfish stuff here when we merge this; a 20th of a stench jelly does not beat a well-used YR

Reply:
Also that'll make it harder to test